packages/docs/docs/docs/architecture.md

```diff
@@ -0,0 +1,102 @@
+---
+id: architecture
+title: Contract Architecture
+---
+
+ZeppelinOS's features such as upgrades, EVM package linking, and vouching, can all be used independently as standalone components using the CLI. However, they can all be tied together when using the CLI with the `--full` option:
+
+```bash
+zos init --full
+```
+
+or
+
+```bash
+zos push --full
+```
+
+When this option is used, your ZeppelinOS project is managed by a set of smart contracts that can be found in [zos/packages/lib/contracts/application](https://github.com/zeppelinos/zos/tree/master/packages/lib/contracts/application). It is important to understand that these contracts are not strictly necessary to work with ZeppelinOS, however, they provide a lot of convenience for projects that intend to become EVM packages or base their entire architecture on ZeppelinOS.
```

Comment:
I would say "Once your project is published, it will be backed by a set of smart contracts that can be found in..."